server/controller/AdminController.js

```diff
@@ -107,6 +105,64 @@ class AdminController {
       });
     }
   }
+
+   /**
+ *@desc disapprove a requests by an admin
+ *
+ * @param {object} req
+ * @param {object} res
+ *
+ * @return {object} the disapproved request
+ */
+static async disapproveARequest(req, res) {
+  try {
+    const { requestId } = req.params;
+    const { requestStatus } = req.body;
+
+    if (!Number.isInteger(Number(requestId))) {
```

Comment:
What is the point of this line of code? Can the application do without it?

Reply:
it is to check if the request Id is an integer